rtime.go

```diff
@@ -26,10 +26,8 @@ var rtime time.Time
 //    }
 //
 func Now() time.Time {
-	var mu sync.Mutex
 	res := make([]time.Time, 0, len(sites))
-	cond := sync.NewCond(&mu)
-	var timedout bool
+	results := make(chan time.Time)
```

Comment:
This probably needs a length of `len(sites)` so that it doesn't deadlock on like 46